fluidBuild.config.cjs

```diff
@@ -213,6 +213,7 @@ module.exports = {
 				"{azure,examples,experimental,packages}/*/*/*/package.json",
 				"{azure,examples,experimental,packages}/*/*/*/*/package.json",
 				"tools/markdown-magic/package.json",
+				"tools/test-tools/package.json",
```

Comment:
Not necessarily for this PR, but is `test-tools` used outside of the client workspace anywhere? If not, can we actually move this package's directory under `packages` somewhere? `markdown-magic` is a bit special, since `docs` also uses it. But I don't think that applies here?

Reply:
Only within client, so we can definitely do this. I think making it its own PR makes sense though, to keep this one easier to review, since there were significant changes inside the package.